io/concatenateHDF5.m

```diff
@@ -0,0 +1,157 @@
+function concatenateHDF5(tifDir,outputDir,hdf5Name,downsmpFactor,plusmazeName,trim)
+
+% Concatenates all tif files in the specified directory into an
+% hdf5 file. The stored files are all downsampled by the factor set in 
+% 'downsmpFactor'. The number of frames to be dropped from the beginning 
+% and end of each trial is set in 'trim'. Bad trials are removed (using the
+% plusmaze text file) and if frames were dropped during a good trial, the 
+% previous frame is used to fill the gap. A new text file is produced
+% indicating the start and stop frames for each trial (as well as the gate
+% up/down frames, the direcitons of the trial and time to complete each
+% trial).
+%
+% Example arguments:
+% tifDir = '/Volumes/COHORT9/cohort9-herrin224/mouse5/day20_ego-right';
+% outputDir = '/Users/jmaxey/Documents/MATLAB/PreFrontal';
+% hdf5Name = 'test.hdf5';
+% downsmpFactor = 0.5;
+% plusmazeName = 'mouse5_day20_ego-right.txt';
+% trim = [10,5];
+%
+% 2015-01-27 Jessica Maxey
+
+outputFileName = fullfile(outputDir,[plusmazeName(1:end-4),'_new.txt']);
+outputFile = fopen(outputFileName,'wt+');
+
+[frame_indices,location_info,time] = getTrialIndicies(fullfile(tifDir,plusmazeName));

```

Comment:
In that case, it should be called `parse_plusmaze` or something to the effect (since it returns more than the indices), and we can deprecate the existing functions.